roles/horizon/tasks/main.yml

```diff
@@ -1,42 +1,26 @@
 ---
-- name: install apache
-  apt: pkg={{ item }}
-  with_items:
-    - apache2
-    - libapache2-mod-wsgi
-
 - name: lesscpy must be in apache PATH
   pip: name=lesscpy version=0.9j
 
 - name: disable apache status
   command: a2dismod status
   notify:
-    - restart apache
+    - reload apache
```

Comment:
This should go into the apache role perhaps. The status module doesn't have much to do with Horizon.